testsuite/features/support/lavanda.rb

```diff
@@ -85,63 +85,63 @@ def init_has_mgrctl
 
   # getter functions, executed on testsuite
   def hostname
-    raise KeyError, 'empty hostname, something wrong' if @in_hostname.empty?
+    raise KeyError, 'empty hostname, something wrong' if @in_hostname.nil? || @in_hostname.empty?
```

Comment:
Nice improvement.

Reply:
thank you

Reply:
```
@in_hostname.nil? || @in_hostname.empty?
```
=>
```
@in_hostname.to_s.empty?
```
maybe shorter? (but less legible, so i'm not sure)

Reply:
I prefer to stick to .nil? evaluation, if you don't mind.
It's easier to understand the reasoning of these conditions, and I put readability on top of performance in this case.